### ISSUE TITLE
Suggestion: amend README

### ISSUE THREAD

**Issue by DutchPete** [tu2-atmanand/RoseRed-ObsidianTheme#4]
It might be worth mentioning that this is a dark theme, otherwise people might be confused if they have Obsidian set to Light.

**Comment by tu2-atmanand** [comment]
Yeah, I think that's what I will have to do now. BTW I was working on having a good gradient for Light theme but nothing seems to be eye appealing to me as of now. 

Where exactly are you suggesting mentioning the dark theme, since I have already mentioned it in the Readme as "A Red Gradient Dark theme for Obsidian" ?
Additionally, if you open the Theme marketplace and try to search for this theme it won't appear since, it's registered in the Obsidian  Plugin Community as a Dark Theme. 

![image](https://github.com/user-attachments/assets/ae177e81-ae09-43d1-b9bd-9374e4c2014a)

 But I also know that, Obsidian gives option to change to Light theme, even if the current theme doesnt support it. I guess thats Obsidian bug, its better to report it on the forum, ill report it soon.

![image](https://github.com/user-attachments/assets/b2a7ceb1-2bcb-43a5-91ed-b00235edddec)

**Comment by DutchPete** [comment]
> Where exactly are you suggesting mentioning the dark theme, since I have already mentioned it in the Readme as "A Red Gradient Dark theme for Obsidian" ?

Oops, my mistake, I missed that, therefore my comment was superfluous, not to say downright silly. My apologies.

> Obsidian gives option to change to Light theme, even if the current theme doesnt support it.

Good point, this never even occurred to me with other similar dark-only themes.

In any case, it is a lovely theme. If you cannot find the right colors and color combinations for a Light theme, why persist? It will take a lot of time and effort and success is not guaranteed  –I admit, that is none of my business 😀

**Comment by tu2-atmanand** [comment]
> In any case, it is a lovely theme. If you cannot find the right colors and color combinations for a Light theme, why persist? It will take a lot of time and effort and success is not guaranteed –I admit, that is none of my business 😀

Thank you so much for sharing your thoughts, I guess then it will be better to keep it as a dark theme only as so many people have already loved it.

**Comment by tu2-atmanand** [comment]
I have made some more improvement, has released the major version, since it's been working fine for most of the components and with most of the plugins. Be in touch if you find any anomaly in the theme.

**Comment by DutchPete** [comment]
> I guess then it will be better to keep it as a dark theme only as so many people have already loved it.

I think that would be best. I must admit, I usually go for Light themes, or in any case themes that have Light versions, but I find your dark theme exceptionally attractive that I use it day and night.

> Be in touch if you find any anomaly in the theme.

I just upgraded it and will let you know if I find anything. Till then, no news is good news, as the saying goes.
